### ISSUE TITLE
第２章代码，代码更新

### ISSUE THREAD

**Issue by jiangyangbo** [xiaobaoonline/pytorch-in-action#3]
如果是新版本，
loss = output.data[0]  应修改为：
loss = output.data.item()

**Comment by jiangyangbo** [comment]
第３章，　103行，105，114行　，存在同样的版本问题
data[0] 应修改为data.item()

**Comment by jiangyangbo** [comment]
第４章，dnn_mnist.py　
82 行，　data[0] 应修改为data.item()

**Comment by jiangyangbo** [comment]
第4章，optim.py
torch_dataset = Data.TensorDataset(data_tensor=x, target_tensor=y)
应修改为
torch_dataset = Data.TensorDataset(x, y)

data[0] 应修改为data.item()
